pyvista/plotting/renderer.py

```diff
@@ -2939,6 +3002,7 @@ def remove_environment_texture(self):
     def close(self):
         """Close out widgets and sensitive elements."""
         self.RemoveAllObservers()
+        self._render_events.clear()
```

Comment:
Thanks for pointing this out... we should do the same for `add_on_render_callback()`'s callbacks on close